gcloud/rest/taskqueue/manager.py

```diff
@@ -50,7 +50,8 @@ def __init__(self, project, taskqueue, task_worker, batch_size=1,
 
         self.stop_event = threading.Event()
 
-        self.tq = TaskQueue(project, taskqueue, creds=service_file)
+        self.tq = TaskQueue(project, taskqueue, creds=service_file,
+                            google_api_lock=google_api_lock)
```

Comment:
Should this be `self.google_api_lock`, as `self.google_api_lock = google_api_lock or threading.RLock()`?

Reply:
Great catch, thanks!